services/processing/YrProcessor/src/Yr/yr_hour.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, String, Float, Integer, TIMESTAMP
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class WeatherHour(Base):
+    __tablename__ = 'weather'
```

Comment:
Kanskje greit å få med hvor dataen kommer fra i navnet
```suggestion
    __tablename__ = 'yr_weather'
```